clang/test/CodeGenHLSL/inline-functions.hlsl

```diff
@@ -61,7 +61,7 @@ unsigned RemoveDupes(unsigned Buf[MAX], unsigned size) {
 }
 
 
-RWBuffer<unsigned> Indices;
+RWBuffer<unsigned> Indices : register(u0);
```

Comment:
Resources without explicit register assignments are not supported yet. If we do not do this then we end up with a handle that is zero initialized, which causes problems.